project/src/main/webapp/scripts/editQuestionOutlinesScripts.js

```diff
@@ -157,13 +151,30 @@ function crop(lx, ly, rx, ry) {
   cropCanvas.drawImage(pdfCanvas, cropX, cropY, cropWidth, cropHeight, 0, 0, cropWidth, cropHeight);
 }
 
+// onclick called when "Add Question" button is clicked
+function submitBoxButton() {
+  const assignmentKey = getUrlVars()['assignment-key'];
+  var qName = document.getElementById('question-name').value;
+  var qPoints = document.getElementById('question-points').value;
+  var qType;
+  if (document.getElementById('multiple-choice').checked){
+      qType="multiple-choice";
+  } else if (document.getElementById('short-answer').checked){
+      qType="short-answer";
+  } else {
+      qType="other";
+  }
 
-// function cropAllPDFs() {
-//     fetch('/manageBox').then(response => response.json()).then(data => {
-
-//         console.log(data);
-
-//         // crop all the pdfs based off the coordinates
-//     });
+  if (lx!=undefined && rx!=undefined && qName!=""){
```

Comment:
Make sure to leave some spacing between variables and operators.

Reply:
implemented